src/rpc/masternode.cpp

```diff
@@ -293,9 +293,9 @@ static std::string GetRequiredPaymentsString(int nBlockHeight, const CDeterminis
             strPayments += ", " + EncodeDestination(dest);
         }
     }
-    if (CSuperblockManager::IsSuperblockTriggered(*governance, nBlockHeight)) {
+    if (CSuperblockManager::IsSuperblockTriggered(*governance, *::triggerman, nBlockHeight)) {
```

Comment:
that's an `rpc` code; you can get access here to NodeContext, can't you?

Reply:
We can and we do give it access via `NodeContext` in f981f1173d1a8faddfc61adf211c57596a356d49, see [relevant](https://github.com/dashpay/dash/blob/f981f1173d1a8faddfc61adf211c57596a356d49/src/rpc/masternode.cpp#L304) [changes](https://github.com/dashpay/dash/blob/f981f1173d1a8faddfc61adf211c57596a356d49/src/rpc/masternode.cpp#L368). It was left out of that commit so keep the focus on `::triggerman` being a smart pointer with a `NodeContext` alias. 

If I have to make `CGovernanceTriggerManager` pass ref by argument, I might as well do it for `CGovernanceManager` and if I'm going to find `CDeterministicMNManager` use while partially deglobalizing it, I might as well add that to the argument list too and that point, it might as well be its own commit where I remove the usage of globals from RPC code, which is exactly what was done.